test/matrix.lean

```diff
@@ -57,6 +57,8 @@ example {a b c d e f g h : α} : ![a, b, c, d, e, f, g, h] 7 = h := by simp
 example {a b c d e f g h : α} : ![a, b, c, d, e, f, g, h] 37 = f := by simp
 example {a b c d e f g h : α} : ![a, b, c, d, e, f, g, h] 99 = d := by simp
 
+-- note for the below: `fin.sum_univ_zero` may go
+
```

Comment:
I don't understand this comment.

Reply:
I worded it terribly, but essentially the simp only will be different from what's written. I'll just update the suggestion next time I'm near a computer